static/js/message_list_view.js

```diff
@@ -1445,4 +1445,76 @@ export class MessageListView {
             message_container.status_message = false;
         }
     }
+
+    update_sticky_recipient_headers() {
```

Comment:
Let's add a comment at the top of this function explaining why it is exists -- which is I think that we always want the display a date at the top of the message feed, and we haven't figured out a way to specify in pure CSS the rule for when the date row should be displayed? Or are there other reasons we need this `.sticky_header` class to be set here?

Reply:
Okay!